x/ccv/provider/keeper/grpc_query_test.go

```diff
@@ -103,3 +105,147 @@ func TestQueryOldestUnconfirmedVsc(t *testing.T) {
 	}
 	require.Equal(t, expectedResult, response.VscSendTimestamp)
 }
+
+func TestQueryConsumerChainOptedInValidators(t *testing.T) {
```

Comment:
Although not specific to this PR, I added some basic tests for some of the other queries.